__docs__/wonder-blocks-dropdown/multi-select.accessibility.mdx

```diff
@@ -3,35 +3,31 @@ import * as MultiSelectAccessibilityStories from './multi-select.accessibility.s
 
 import {OptionItem, MultiSelect} from "@khanacademy/wonder-blocks-dropdown";
 import {View} from "@khanacademy/wonder-blocks-core";
-import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+import {LabeledField} from "@khanacademy/wonder-blocks-labeled-field";
 
 <Meta of={MultiSelectAccessibilityStories} />
 
 export const MultiSelectAccessibility = () => (
     <View>
-        <LabelLarge
-            tag="label"
-            id="label-for-multi-select"
-            htmlFor="unique-multi-select"
-        >
-            Associated label element
-        </LabelLarge>
-        <MultiSelect
-            aria-labelledby="label-for-multi-select"
-            id="unique-multi-select"
-            selectedValues={["one"]}
-        >
-            <OptionItem
-                label="First element"
-                aria-label="First element, selected"
-                value="one"
-            />
-            <OptionItem
-                label="Second element"
-                aria-label="Second element, unselelected"
-                value="two"
-            />
-        </MultiSelect>
+        <LabeledField
+            label={<strong>Associated label element</strong>}
```

Comment:
Do we need the label wrapped in the `<strong>` tag? I'm wondering if we should use the default styling in our examples so we can avoid one-offs!

Reply:
We probably don't need it! I was trying to make it match how it looked before.

Reply:
Do these examples even get used somehow? I don't think they are actually rendered. Maybe something broke along the way? I'm considering removing them since the examples come from the `*.accessibility.stories` files.

Reply:
I'm also a bit unsure when we should create both `.accessibility.mdx` files and `.accessibility.stories.ts` files. I'm okay with combining them and keeping the docs in `.accessibility.stories.ts` files instead!

cc: @jandrade in case you have thoughts on this!